examples/carousel/createCarousel.md

```diff
@@ -0,0 +1,30 @@
+# Sending a message to all users in all chats for one event.
+Do you want to send a message to all users going to a specific event? Then you can use the sendToAllChatsForEvent() operation.
```

Comment:
Maybe this text should change? It is about creating the carousel, not sending a message. The one below (the lookup one) should also change.